src/client.cpp

```diff
@@ -527,8 +527,8 @@ ProcessResponseEnum HttpClient::ProcessResponse(Value& result, bool notification
 bool TcpClient::GenerateHeader()
 {
     log_trace();
-    header_ << request_.Length() << ":";
-    log_debug("Request length=" << request_.Length());
+    header_ << static_cast<uint64_t>(request_.Length()) << ":";
```

Comment:
It's definitely C++11. If you want to keep backwards compatibility this is not an option.